backend/app/models/collection.py

```
@@ -1,32 +1,18 @@
 from uuid import UUID, uuid4
 from datetime import datetime
-from typing import Optional
+from typing import Any, List, Optional
```

Ruff (0.13.3)

`typing.List` is deprecated, use `list` instead

(UP035)